docs/installation.md

```diff
@@ -72,34 +72,43 @@ If you want to help develop Lychee, install the development dependencies by remo
 
 ### Configuration
 
+After installing Lychee, some additional configuration is necessary as detailed out below.
+
 #### Public Directory
-After installing Lychee, configure your web server's root to be the `public` directory. The `index.php` in this directory serves as the interface for all HTTP requests to Lychee's API. Note that the `.env` file should **not** be exposed online as it contains the encryption key used for cookies as well as database credentials.
 
-#### Configuration Files
-All of the configuration files for Lychee are stored in the `config` directory. Each option is documented, so feel free to look through the files and get familiar with the options available to you.
+Configure your web server's root to be the `public` directory. The `index.php` in this directory serves as the interface for all HTTP requests to Lychee's API.
 
 #### Directory Permissions
-After installing Lychee, you need to configure some permissions. Directories within the `storage` and the `bootstrap/cache` directories should be writable by your web server or Lychee will not run. Additionaly `public/uploads`, `public/dist` and `public/sym` need to be writable to upload files, customise the CSS and utilise [symbolic links](https://lycheeorg.github.io/docs/settings.html#symbolic-link) respectively.
 
-#### Copy configuration file
-If you have not copied the `.env.example` file to a new file named `.env`, you should do that now.
+You need to configure some file and directory permissions. Directories within the `storage` and the `bootstrap/cache` directories should be writable by your web server or Lychee will not run. Additionally, `public/uploads`, `public/dist` and `public/sym` need to be writable to upload files, customise the CSS and utilise [symbolic links](https://lycheeorg.github.io/docs/settings.html#symbolic-link) respectively.
+
+#### Application Settings
+
+The main directory should contain a `.env` file.
+Normally, Composer has generated this file for you based on `.env.example` while Composer has been installing the dependencies (see above).
+If the file `.env` does not exist, then you should copy `.env.example` to a new file named `.env` now.
+Note that the `.env` file should **not** be exposed online as it contains the encryption key used for cookies as well as database credentials.
 
-#### Application Key
-The next thing you should do after installing Lychee is set your application key to a random string. This is easily done by using the `php artisan key:generate` command.
+##### Application Key
 
-If you don't want to use this command and set the key yourself, this string should be 32 characters long and can be set in the `.env` environment file.
+The `.env` file contains the setting `APP_KEY` which holds your application key.
+Use the `php artisan key:generate` command to set the application key to a random value.
```

Comment:
This is also done by composer, but repeating it does no harm.

Reply:
I wonder more about the 32 characters. I used this command and in my case the result was a string starting with `base64:...` and then followed by 48 characters. I am not sure what happens, if you only use 32 characters or if you get low entropy, because you cannot express a truly random string with printable characters only.

Do we really want to recommend users to create their own random string?